MASFoundation/Classes/_private_/services/network/MASNetworkingService.m

```diff
@@ -1896,7 +1903,7 @@ - (void)cancelAllRequests
 
 
 - (MASURLRequest*)getURLRequest:(NSString *)httpMethod endPoint:(NSString *)endPoint parameters:(NSDictionary *)parameterInfo headers:(NSDictionary *)headerInfo requestType:(MASRequestResponseType)requestType responseType:(MASRequestResponseType)responseType isPublic:(BOOL)isPublic
-timeoutInterval:(NSTimeInterval)timeoutInterval
+                timeoutInterval:(NSTimeInterval)timeoutInterval
 {
     //
     // Update the header
```

Comment:
This file is showing lot of changes due to white space addition or deletion. Please revert back the changes and add only relevant changes.